src/lesson/lesson01.js

```diff
@@ -262,6 +262,170 @@ Lesson01ExerciseStepPlayer.prototype = {
   },
 };
 
+function Lesson01ExerciseStepPlayerMulti(isExample, nGames, nRowsList,
+                                         nColsList, robotPositionList,
+                                         obstaclePositionsList) {
+  Lesson.LessonStepPlayer.call(this);
+  this.lessonStepPlayers = Array(nGames);
```

Comment:
It's surprising this works without "new", but I just found out it's indeed standard.